dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/blobstore.properties

```diff
@@ -1,4 +1,4 @@
-objectstore.implementation=swift
+implementation=object-storage
```

Comment:
it's not consistent with "objectstore" just below

Reply:
> it's not consistent with "objectstore" just below

Looking at the final diff, I do not see any mention to `objectstore`.

Could you develop?